src/main/webapp/app/trial/trial.component.html

```diff
@@ -30,6 +30,8 @@
             <span id="nctId" style="margin-left:10px">
                 <a target="_blank" href="https://clinicaltrials.gov/ct2/show/{{item.nct_id}}">{{ item.nct_id }}</a>
             </span>
+            <button class="btn btn-primary" (click)="loadMongo()" style="margin-left: 10px;">Load it to Mongo DB</button>
```

Comment:
This is too technical for the curator. Probably `Finish curation` would be better?